index.js

```diff
@@ -12,6 +12,7 @@ const fs = require("fs"),
 require('dotenv').config();
 
 const isProduction = process.env.NODE_ENV === "production";
+const isTest = process.env.NODE_ENV === "test";
```

Comment:
is the variable isTest necessary, I don't see anywhere you are making use of it.